examples/Accelerators/DarcyAccelerated/darcy_singletrial.jl

```diff
@@ -0,0 +1,241 @@
+
+    smoothness = 1.0
+    corr_length = 0.25
+    dofs = 50
+
+    grf = GRF.GaussianRandomField(
+        GRF.CovarianceFunction(dim, GRF.Matern(smoothness, corr_length)),
+        GRF.KarhunenLoeve(dofs),
+        pts_per_dim,
+        pts_per_dim,
+    )
+
+    # We define a wrapper around the GRF, and as the permeability field must be positive we introduce a domain constraint into the function distribution. Henceforth, the GRF is interfaced in the same manner as any other parameter distribution with regards to interface.
+    pkg = GRFJL()
+    distribution = GaussianRandomFieldInterface(grf, pkg) # our wrapper from EKP
+    domain_constraint = bounded_below(0) # make κ positive
+    pd = ParameterDistribution(
+        Dict("distribution" => distribution, "name" => "kappa", "constraint" => domain_constraint),
+    ) # the fully constrained parameter distribution
+
+    # Now we have a function distribution, we sample a reasonably high-probability value from this distribution as a true value (here all degrees of freedom set with `u_{\mathrm{true}} = -0.5`). We use the EKP transform function to build the corresponding instance of the ``\kappa_{\mathrm{true}}``.
+    u_true = -1.5 * ones(dofs, 1) # the truth parameter
+    κ_true = transform_unconstrained_to_constrained(pd, u_true) # builds and constrains the function.  
+    κ_true = reshape(κ_true, N, N)
+
+    # Now we generate the data sample for the truth in a perfect model setting by evaluating the the model here, and observing it by subsampling in each dimension every `obs_ΔN` points, and add some observational noise
+    darcy = Setup_Param(pts_per_dim, obs_ΔN, κ_true)
+    h_2d = solve_Darcy_2D(darcy, κ_true)
+    y_noiseless = compute_obs(darcy, h_2d)
+    obs_noise_cov = 0.05^2 * I(length(y_noiseless)) * (maximum(y_noiseless) - minimum(y_noiseless))
+    truth_sample = vec(y_noiseless + rand(rng, MvNormal(zeros(length(y_noiseless)), obs_noise_cov)))
+
+    # Now we set up the Bayesian inversion algorithm. The prior we have already defined to construct our truth
+    prior = pd
+
+    # We define some algorithm parameters, here we take ensemble members larger than the dimension of the parameter space
+    N_ens = dofs + 2    # number of ensemble members
+    N_iter = 20         # number of EKI iterations
+    N_trials = 5       # number of trials 
+
+    N_iter = 10 # number of EKI iterations
+    initial_params = construct_initial_ensemble(rng, prior, N_ens)
+    eki_trad = EKP.EnsembleKalmanProcess(initial_params, truth_sample, obs_noise_cov, Inversion())#,scheduler = DataMisfitController(on_terminate = "continue"))
+    eki_acc = EKP.EnsembleKalmanProcess(
+        initial_params,
+        truth_sample,
+        obs_noise_cov,
+        Inversion(),
+        accelerator = NesterovAccelerator(),
```

Comment:
add a `scheduler = deepcopy(scheduler)` keyword here. and for the `eki_trad` 

Allow the user to toggle between `DefaultScheduler(0.5)` and the `DataMisfitController(terminate_at=1e4)`